src/smexperiments/experiment.py

```diff
@@ -223,3 +224,31 @@ def create_trial(self, trial_name=None, trial_name_prefix="SageMakerTrial"):
             experiment_name=self.experiment_name,
             sagemaker_boto_client=self.sagemaker_boto_client,
         )
+
+    def delete_all(self, action):
+        """
+        Force to delete the experiment and associated trials, trial components under the experiment.
+
+        Args:
+            action (str): pass in string 'force--' to confirm recursively delete all the experiments, trials,
```

Comment:
Why not '--force' (kind of mimicing command line tools), or just 'force'?

Reply:
ya, changed to "--force"

Reply:
nit - the "force" parameter seems superfluous. The user is writing code to call this method - its not like they are accidentally clicking a button.